internal/string.h

```diff
@@ -30,6 +30,7 @@ enum ruby_rstring_private_flags {
 #endif
 
 /* string.c */
+VALUE rb_yjit_str_dup(VALUE str);
```

Comment:
```suggestion
```

I think YJIT can use the function without this declaration. bindgen alone should be enough.

Reply:
Oh but bindgen needs to read this declaration 😅 nvm

Reply:
TIL

Reply:
Sorry that I confused you, but you need to do either of these:

1. Declare `rb_str_dup_m` here and let bindgen use it
2. Or just declare that yourself in Rust code

Please rerun `make yjit-bindgen` if you take the (1) route.